command-line-options.md

```diff
@@ -0,0 +1,203 @@
+        -m                      merge text reconstitution files
+        -H                      merge hist reconstitution files
+
+standalone maintenance flags for use with -i
+        -r text/metrics         repair text inventory
+        -r text/changelog       repair text datastore
+        -r hist/metrics         repair hist inventory
+        -r hist/<period>        repair hist rollups
+        -j                      only write journal data to otther nodes
+
+optional behavior flags:
+        -c <file>               load config from <file> (full path)
+                                        default: /opt/circonus/etc/snowth.conf
+        -d                      debugging
+        -D                      foreground operations (don't daemonize)
+        -u <user>               run as <user>
+        -g <group>              run as <group>
+        -t <path>               chroot to <path>
+        -l <logname>            enable <logname>
+        -L <logname>            disable <logname>
+
+reconstitute parameters:
+        -B                      Reconstitute mode
+        -T <topo_hash>          Reconstitute from remote topology
+        -O <ip>[:<port>]        Reconstitute from remote host
+        -A [<type>[,<type>]]    Reconstitute type
+                                Acceptable values: nnt,text,hist,raw,metrics
+                                All if omitted
+        -E                      Reset mode - remove existing state files, start
+from beginning
+
+this usage message:
+        -h                      usage
+```
+
+## Process Control Options
+
+ * `-k <start|stop|status>`
+
+`status` will exit `0` if the process is running, non-zero otherwise.
+
+## Operating Mode Options
+
+These options are mutually exclusive of one another. One or the other is
+required.
+
+ * `-i <uuid>`
+
+Identify this node with `<uuid>`.  This is the normal mode of operation.
+
+ * `-e`
+
+Boot the node in ephemeral mode. Ephemeral nodes are read-only participants in
+the cluster. They do not appear in the cluster topology, and do not accept
+incoming metrics, but may be used to read metric data from other nodes and
+perform intensive computation that would add unreasonable load to the main
+nodes.
+
+## Loader Options
+
+**DEPRECATED**
+
+These options imply foreground operation and perform a specific task, then
+exit. They are only valid in identified mode (`-i`).
+
+ * `-m` 
+
+Merge text reconstitution files.
+
+ * `-H`
+
+Merge histogram reconstitution files. 
+
+These options were used in a previous version of the reconstitute process and
+are no longer strictly required. They may be removed in a future version.
+
+## Maintenance Options
+
+These options imply foreground operation and perform a specific task, then
+exit. They are only valid in identified mode (`-i`).
+
+ * `-r text/metrics`
+
+Repair text inventory.
+
+ * `-r text/changelog`
+
+Repair text datastore.
+
+ * `-r hist/metrics`
+
+Repair histogram inventory.
+
+ * `-r hist/rollups`
```

Comment:
There's no command for "rollups" - you have to specify the rollup itself. IE:

`-r hist/60` or `-r hist/300`

Reply:
I based that off of the existing `snowthd -h` output.  I will clarify here and in the upcoming PR for fixing the usage summary.